simulations/vip-469/bsctestnet.ts

```diff
@@ -0,0 +1,179 @@
+import { checkInterestRate } from "src/vip-framework/checks/interestRateModel";
+
+import vip469, {
+  BTCB_PRIME_CONVERTER,
+  BaseAssets,
+  COMPTROLLER_LIQUID_STAKED_BNB_POOL,
+  CONVERSION_INCENTIVE,
+  ETH_PRIME_CONVERTER,
+  RISK_FUND_CONVERTER,
+  USDC_PRIME_CONVERTER,
+  USDT_PRIME_CONVERTER,
+  XVS_VAULT_CONVERTER,
+  market,
+  token,
+} from "../../vips/vip-469/bsctestnet";
+import POOL_REGISTRY_ABI from "./abi/PoolRegistry.json";
+import RESILIENT_ORACLE_ABI from "./abi/ResilientOracle.json";
+import SINGLE_TOKEN_CONVERTER_ABI from "./abi/SingleTokenConverter.json";
+import COMPTROLLER_ABI from "./abi/comptroller.json";
+import VTOKEN_ABI from "./abi/vToken.json";
+
+const { POOL_REGISTRY, NORMAL_TIMELOCK, RESILIENT_ORACLE } = NETWORK_ADDRESSES["bsctestnet"];
+const BLOCKS_PER_YEAR = BigNumber.from("10512000");
+
+forking(49227951, async () => {
+  const provider = ethers.provider;
+  const oracle = new ethers.Contract(RESILIENT_ORACLE, RESILIENT_ORACLE_ABI, provider);
+  const poolRegistry = new ethers.Contract(POOL_REGISTRY, POOL_REGISTRY_ABI, provider);
+  const comptroller = new ethers.Contract(COMPTROLLER_LIQUID_STAKED_BNB_POOL, COMPTROLLER_ABI, provider);
+  const vTokenContract = new ethers.Contract(market.vToken.address, VTOKEN_ABI, provider);
+
+  describe("vTokens deployment", () => {
+    it(`should deploy market`, async () => {
+      await checkVToken(market.vToken.address, market.vToken);
+    });
+  });
+
+  testVip("vip469", await vip469());
+
+  describe("Post-VIP behavior", () => {
+    describe("Oracle configuration", async () => {
+      it(`has the correct ${token.symbol} price`, async () => {
+        const price = await oracle.getPrice(token.address);
+        expect(price).to.be.eq(parseUnits("2569.216740763257922116", 18));
+      });
+    });
+
+    describe("PoolRegistry state", () => {
+      it(`should add ${market.vToken.symbol} to the Comptroller`, async () => {
+        const poolVTokens = await comptroller.getAllMarkets();
+        expect(poolVTokens).to.contain(market.vToken.address);
+      });
+
+      it(`should register ${market.vToken.symbol} in PoolRegistry`, async () => {
+        const registeredVToken = await poolRegistry.getVTokenForAsset(
+          COMPTROLLER_LIQUID_STAKED_BNB_POOL,
+          market.vToken.underlying.address,
+        );
+        expect(registeredVToken).to.equal(market.vToken.address);
+      });
+    });
+
+    describe("Risk parameters", () => {
+      checkRiskParameters(market.vToken.address, market.vToken, market.riskParameters);
+
+      it("should pause borrowing", async () => {
+        expect(await comptroller.actionPaused(market.vToken.address, 2)).to.equal(true);
+      });
+
+      it(`should have a protocol seize share ${market.riskParameters.protocolSeizeShare}`, async () => {
+        expect(await vTokenContract.protocolSeizeShareMantissa()).to.equal(market.riskParameters.protocolSeizeShare);
+      });
```

Comment:
This (`protocolSeizeShareMantissa`) is checked twice in the simulation. No problem for me due to special case, but maybe you prefer to remove one of it :)